.github/workflows/tests.yml

```
@@ -56,7 +56,7 @@ jobs:
           key: style-cache-${{ matrix.stable && 'stable-' || 'master-' }}${{ github.sha }}
           restore-keys: style-cache-${{ matrix.stable && 'stable-' || 'master-' }}
 
-      - run: brew test-bot --only-tap-syntax
+      - run: brew test-bot --only-tap-syntax ${{ matrix.stable && '--stable' }}
```

```suggestion
      - run: brew test-bot --only-tap-syntax ${{ matrix.stable && '--stable' || '' }}
```

(otherwise it will output `brew test-bot --only-tap-syntax false`)

---

Thanks. 😵‍💫